sample/MinimalSample/Program.cs

```diff
@@ -0,0 +1,10 @@
+using LinkDotNet.NCronJob;
+
+var builder = Host.CreateApplicationBuilder(args);
+
+builder.Services.AddJob((ILogger<Program> logger, TimeProvider timeProvider) =>
```

Comment:
Really nice - I thought about the naming and whether or not that clashes with other libraries. The only example I could find was: `Foundatio.Extensions.Hosting.Jobs`. So that should be fine then.